src/c++/perf_analyzer/command_line_parser.cc

```diff
@@ -373,6 +374,13 @@ CLParser::Usage(const std::string& msg)
              "that the average latency is used to determine stability",
              18)
       << std::endl;
+  std::cerr << FormatMessage(
+                   " --serial-sequences: Enables serial sequence mode "
+                   "where only 1 request is live for a sequence and ensures "
+                   "that sequences are issued in a nearly round robin fashion. "
```

Comment:
remove mention of round robin. Say a maximum of one request outstanding at a time for any given sequence.